doc/lsst.ctrl.bps.htcondor/userguide.rst

```diff
@@ -321,6 +321,84 @@ existing file.
 
 .. __: https://htcondor.readthedocs.io/en/latest/codes-other-values/glossary.html#term-Glidein
 
+.. _htc-plugin-releasing:
+
+Releasing held jobs
+-------------------
+
+Occasionally, when HTCondor encounters issues during job's execution it places
+the job in the hold state. You can see what jobs you submitted are being
+currently held and why by using the command:
+
+.. code-block::
+
+   condor_q -held
+
+If any of your jobs are being held, it will display something similar to::
+
+    -- Schedd: sdfrome002.sdf.slac.stanford.edu : <172.24.33.226:21305?... @ 10/02/24 10:59:41
+    ID           OWNER  HELD_SINCE  HOLD_REASON
+    5471792.0    jdoe   7/10 08:27  File '/sdf/group/rubin/sw/conda/envs/lsst-scipipe-8.0.0/bin/condor_dagman' is missing or not executable
+    7636239.0    jdoe   3/20 01:32  Job raised a signal 11. Handling signal as if job has gone over memory limit.
+    5497548.0    jdoe   3/6  00:14  Job raised a signal 9. Handling signal as if job has gone over memory limit.
+
+.. note::
+
+   If you would like to display held jobs that were submitted for execution
+   by other users, use ``condor_q -held <username>`` instead where
+   ``<username>`` is the user account which held jobs you would like to check.
+   See `condor_q`_ man page for other supported options.
+
+The job that is in the hold state can be released from it with
+`condor_release`_ providing the issue that made HTCondor put it in this state
+has been resolved. For example, if your job with id 1234.0 was placed in the
+hold state because during the execution it exceeded 2048 MiB you requested for
+it during the submission, you can double the amount of memory it should request with
+
+.. code-block::
+
+   condor_qedit 1234.0 RequestMemory=4096
+
+and than release it from the hold state with
+
+.. code-block::
+
+   condor_release 1234.0
```

Comment:
What does condor_release of a job mean?  Probably want to be explicit that it means something like HTCondor puts the job into the IDLE state and will rerun the job using the exact same command and environment as before.

Reply:
Don't know if we want to say something like the "No such file or directory" typically happen with a glidein expiration or a filesystem issue.  After making sure allocateNodes is running (see bps automatic provisioning for future submission), a job typically runs ok after release.

Reply:
Speaking of bps automatic provisioning, you had mentioned you had discovered provisioningMaxIdleTime having an incorrect value in the text describing the default.  Do you want to go ahead and fix that text in this ticket since updating the documentation?

Reply:
Yes, I'll fix the description of the defaults.